docs/environment_creation_from_scratch.md

```diff
@@ -1,24 +1,21 @@
 # Creating a new environment from scratch
 
-TODO: Update this once splitting out secrets and app roles is done, add instructions for sorting those and things to update in meta.
-
-When setting up a new environment;
+When setting up a new environment:
 1. DNS records will need to be created by DLUHC for certificate validation and to point at the cloudfront distribution and app load balancer
 1. Secret values will need to be filled in manually
-1. Need to set up deployments etc.
-1. Need to ensure anything that needs adding to the meta module (e.g. for roles access to ecr) is done, since some of that is currently manual
+1. The meta account will need updating with additional roles access to the ecr, since this is currently manual
+1. A new deployment pipeline will need to be setup
 
 ## The process
 
 ### Run an initial apply
 
-```terraform apply -target="module.networking" -target="module.front_door" -var="initial_create=true"```
+```terraform apply -target="module.networking" -target="module.front_door" -target="module.certificates" --target="module.application_roles" --target="module.application_secrets" var="initial_create=true"```
```

Comment:
I think this was an unnecessary change by me - terraform should pick up the dependencies of the `application_secrets` module (it needs a part of the `application_roles` module) and create them as necessary, rather than us commanding it create the whole module. I've updated the command to remove this 😀